utils.py

```diff
@@ -595,4 +630,62 @@ def zipdir(path, ziph):
     # Write zip as a result to last trial for now
     if writeToDB:
         post_file_to_trial(session_zip,dynamic_ids[-1],tag='session_zip',
-                           device_id='all')    
\ No newline at end of file
+                           device_id='all')    
+    
+        
+
+def plot_subplots_with_shading(mean_df, sd_df, columns=None, leg=None):
```

Comment:
@suhlrich there is a utilsPlotting script already. It would be great to leverage what is there already instead of re-creating something specific to a use case. I think the ultimate goal is to have a great plotting tool with many functionalities rather than multiple plotting functions. I believe there was already support for most of what this function is doing.

Reply:
ok I moved it over to utilsPlotting. It is different from the function that was already there, so I made it nice. It is pretty generally useful.